frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/utils.ts

```diff
@@ -16,15 +17,28 @@ export enum FAILED_RETURN_TYPE {
   UPLOAD = "UPLOAD",
   VALIDATION = "VALIDATION",
   CONVERSION = "CONVERSIOn",
+  PROCESS = "PROCESS",
 }
 
 export type FailReturn = {
   isFailed: boolean;
-  error?: VALIDATION_STATUS | UPLOAD_STATUS | CONVERSION_STATUS;
+  error?:
+    | VALIDATION_STATUS
+    | UPLOAD_STATUS
+    | CONVERSION_STATUS
+    | PROCESSING_STATUS;
   type?: FAILED_RETURN_TYPE;
 };
 
 export function checkIfFailed(datasetStatus: DatasetUploadStatus): FailReturn {
+  if (datasetStatus.processing_status === PROCESSING_STATUS.FAILURE) {
+    return {
+      error: PROCESSING_STATUS.FAILURE,
+      isFailed: true,
+      type: FAILED_RETURN_TYPE.PROCESS,
+    };
+  }
+
```

Comment:
If the processing status failure's message is vague, do we want to prioritize others in the case of multiple failures?

Reply:
Ooh that's a really good point 😆 Thanks for pointing that out, Seve! 🙏 DONE!